app/views/collections/index.html.erb

```diff
@@ -1,42 +1,30 @@
 <p id="notice"><%= notice %></p>
-
 <h1>Collections</h1>
+<%= link_to 'New Collection', new_collection_path, class: "btn btn-primary pull-right collection-show-buttons" %>
+<hr style="border: 1px solid darkgrey;">
 
-<%= link_to 'New Collection', new_collection_path, class: "btn btn-primary" %>
-
-<div class="panel panel-primary">
-  <div class="panel-heading">
-    <h3 class="panel-title">All collections</h3>
-  </div>
-  <div class="panel-body">
-    <table class="table table-striped">
-      <thead>
-        <tr>
-          <th>Name</th>
-          <th>Slides</th>
-          <th colspan="3"></th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <% @collections.each do |collection| %>
-          <% if collection.name != "generic" %>
-            <tr>
-              <td><%= link_to collection.name, collection %></td>
-              <td>
-                <%= collection.slides.count.to_s %>
-              </td>
-              <td><%= link_to 'Edit', edit_collection_path(collection) %></td>
-              <td><%= link_to 'Destroy', collection, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-            </tr>
-          <% end %>
+<h3>All collections</h3>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Slides</th>
+        <th colspan="3"></th>
```

Comment:
This pattern should include a header for this column, like `Actions` and the links should be in a `<ul>` in one column